src/XrdPfc/XrdPfcIO.hh

```diff
@@ -85,6 +86,15 @@ private:
    int    m_active_prefetches {0};
    bool   m_allow_prefetching {true};
    bool   m_in_detach         {false};
+
+   int                m_incomplete_count {0};
+   std::map<int, int> m_error_counts;
+   bool register_incomplete_read() {
```

Comment:
I mean to put `protected:` above this line, then only the new interfaces are protected, and the new and pre-existing members can remain private.

Reply:
Done, moved it down.